app/views/ad_mailer/send_token.text.erb

```diff
@@ -0,0 +1,12 @@
+Poštovani,
+
+Na vaš zahtjev šaljemo vam link za <%= @action %> vašeg oglasa:
+<%= @url %>
+
+Molimo vas da ne šaljete ovaj mail dalje jer svatko tko ima ovaj link može mijenjati vaš oglas.
+
+Ovaj e-mail je automatski poslan zato što ste vi ili netko drugi zatražio uređivanje oglasa objavljenog na https://www.potres-petrinja.hr/. Ukoliko to niste bili vi slobodno zanemarite ovaj e-mail.
```

Comment:
Like @vfonic suggested, maybe capitalise Vi and Vas. I forgot about that when writing this text.

Reply:
Done https://github.com/vlado/earthquake-croatia/pull/110/commits/1572bcac2b9c6b34e0d709ffdbe758f4ebe58af1